src/booster.py

```diff
@@ -0,0 +1,84 @@
+import shutil
+import subprocess
+from pathlib import Path
+
+import typer
+import yaml
+
+app = typer.Typer()
+
+BASE_DIR = Path(__file__).resolve().parent.parent
+OUTPUT_DIR = BASE_DIR / "output"
+DEFAULT_CONFIG = BASE_DIR / "sample_config.yml"
+
+
+def load_config(config_path: Path):
+    """Load project settings from YAML configuration file."""
+    with open(config_path, "r") as file:
+        config = yaml.safe_load(file)
+    return config
+
+
+def django_start_project(config):
+    """Create a Django project using the specified configuration."""
+    project_name = config["project_name"]
+    template_path = config.get("template_path", "template/")
+    output_dir = Path(config.get("output_dir")) or OUTPUT_DIR
+    output_dir.mkdir(parents=True, exist_ok=True)
+    project_path = output_dir / project_name
+    if project_path.exists():
+        typer.echo(f"Directory '{project_path}' already exists.")
+        if typer.confirm(
+            "Do you want to clear this directory and start fresh?", default=False
+        ):
+            # Delete the existing directory
+            shutil.rmtree(project_path)
+            typer.echo(f"Cleared the directory: {project_path}")
+        else:
+            typer.echo("Aborting project creation.")
+            raise SystemExit(1)
+    project_path.mkdir(parents=True, exist_ok=True)
+
+    # Call django-admin startproject
+    typer.echo(f"Starting Django project '{project_name}' in {project_path}...")
+
+    try:
+        subprocess.run(
+            [
+                "django-admin",
+                "startproject",
+                project_name,
+                str(project_path),
+                "--template",
+                template_path,
+                "--extension",
+                "py,env,sh,toml,yml",
+                "--exclude",
+                "nothing",
+            ],
+            check=True,
+        )
+        typer.echo(f"Django project '{project_name}' created successfully!")
+    except subprocess.CalledProcessError:
+        typer.echo(f"Failed to create the project '{project_name}'.", err=True)
+        raise SystemExit(1)
+
+
+@app.command()
+def fire(
+    config_path: Path = typer.Option(
+        DEFAULT_CONFIG,
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Make this a mandatory argument

The script cannot proceed without a config file. A default for the name of a project for example is never going to be reasonable. So might as well make it a mandatory argument.